cfme/intelligence/reports/reports.py

```diff
@@ -364,6 +364,8 @@ def __init__(self, path_to_report, datetime, candu=False, appliance=None):
         self.path = path_to_report
         self.datetime = datetime
         self.candu = candu
+        self.datetime_in_tree = version.pick({"5.6": self.datetime,
```

Comment:
@RonnyPfannschmidt 
we do not support versions less than 5.6, so 5.6 is equal to lowest.